x-pack/plugins/security_solution/server/features.ts

```diff
@@ -296,18 +296,24 @@ const subFeatures: SubFeatureConfig[] = [
         groupType: 'mutually_exclusive',
         privileges: [
           {
-            api: [`${APP_ID}-writeTrustedApplications`, `${APP_ID}-readTrustedApplications`],
+            api: [
+              'lists-all',
+              'lists-read',
+              'lists-summary',
+              `${APP_ID}-writeTrustedApplications`,
+              `${APP_ID}-readTrustedApplications`,
+            ],
             id: 'trusted_applications_all',
             includeIn: 'none',
             name: 'All',
             savedObject: {
-              all: [],
+              all: ['exception-list-agnostic'],
```

Comment:
Should we use `EXCEPTION_LIST_NAMESPACE_AGNOSTIC` from the `kbn-securitysolution-list-constants` package?